### PR TITLE
better landmark docs

### PR DESCRIPTION
**Pull request by karussell**
Please review and also waiting for #960 being merged ...

### CODE REVIEW

File: docs/core/landmarks.md [hunk 1]
```diff
@@ -10,3 +10,15 @@ for (bidirectional) A* and are done. It is important that although a weight
 **approximation** is used the landmarks algorithm is still not heuristic per
 default. You can make it faster and more heuristical behaving when you set
 the epsilon parameter lower than 1.
+
+For the world wide case we limit the routing in a certain way for landmarks.
+You can only route within the EU, Africa and Asia. The other algorithms are
```
Comment (boldtrn) [inline docs/core/landmarks.md]: For routing requests using the landmark algorithm it is not possible to cross the borders between EU, Africa, and Asia?

Might be a bit more understandable? Or generally stating that you cannot route accross these borders using the landmark algorithm. The above sounds like routing in USA wouldn't be possible at all.

File: docs/core/landmarks.md [hunk 2]
```diff
@@ -10,3 +10,15 @@ for (bidirectional) A* and are done. It is important that although a weight
...
+not affected from this limitation though. For landmarks this is important
```
Comment (boldtrn) [inline docs/core/landmarks.md]: affected **by**? Not sure though

File: docs/core/landmarks.md (hunk 1 above)
Comment (karussell) [inline docs/core/landmarks.md]: Ah, ups. You are right.

**Comment by karussell** [comment]
Will correct the problems and also add the new prepare.lm.threads config possibility.

**Pull request merged**